test/e2e/testdrivers/1.24.yaml

```diff
@@ -36,7 +36,6 @@ DriverInfo:
     volumeLimits: true
     singleNodeVolume: true
     topology: true
-    capacity: true
```

Comment:
Just to confirm: did you check if any new options were added to the upstream 1.24 configuration struct?

Reply:
Thanks for the hint. There are two options that we are not using atm.
```
	CapVolumeMountGroup   Capability = "volumeMountGroup"   // Driver has the VolumeMountGroup CSI node capability. Because this is a FSGroup feature, the fsGroup capability must also be set to true.

	// The driver publishes storage capacity information: when the storage class
	// for dynamic provisioning exists, the driver is expected to provide
	// capacity information for it.
	CapCapacity Capability = "capacity"
```
Do we want to enable these capabilities?

Reply:
Looking at the corresponding CSI documentation pages for [CSI Driver fsGroup Support](https://kubernetes-csi.github.io/docs/support-fsgroup.html) and [Storage Capacity Tracking](https://kubernetes-csi.github.io/docs/storage-capacity-tracking.html), it looks to me like we'd need to do some work to support these features. I think it's fine to keep them disabled for now.

Would you mind setting them to `false` explicitly to serve as a future note/reminder for us what the level of support for our driver currently is?